nilearn/signal.py

```diff
@@ -429,39 +431,39 @@ def butterworth(
             signals = output
         else:
             signals[...] = output
+    elif copy:
```

Comment:
technically a correct refactoring but maybe not as easy to read for a human being